CoffeeChat/Controller/SchedulingPlacesViewController.swift

```diff
@@ -0,0 +1,289 @@
+        view.addSubview(locationsCollectionView)
+
+        nextButton.setTitle("Finish", for: .normal)
+        nextButton.layer.cornerRadius = LayoutHelper.shared.getCustomVerticalPadding(size: 27)
+        nextButton.isEnabled = false
+        nextButton.setTitleColor(.white, for: .normal)
+        nextButton.titleLabel?.font = ._20CircularStdBold
+        nextButton.backgroundColor = .inactiveGreen
+        nextButton.addTarget(self, action: #selector(nextButtonPressed), for: .touchUpInside)
+        view.addSubview(nextButton)
+
+        backButton.setTitle("Go back", for: .normal)
+        backButton.backgroundColor = .clear
+        backButton.setTitleColor(.secondaryOlive, for: .normal)
+        backButton.titleLabel?.font = ._16CircularStdBook
+        backButton.addTarget(self, action: #selector(backButtonPressed), for: .touchUpInside)
+        view.addSubview(backButton)
+
+        // TODO: Replace with networking when available
+        let campusLocations = [
+          "Atrium Cafe",
+          "Cafe Jennie",
+          "Gimme Coffee",
+          "Goldie's Cafe",
+          "Green Dragon",
+          "Libe Cafe",
+          "Mac's Cafe",
+          "Martha's Cafe",
+          "Mattin's Cafe",
+          "Temple of Zues"
+        ]
+        let ctownLocations = [
+          "Kung Fu Tea",
+          "Starbucks",
+          "Mango Mango",
+          "U Tea"
+        ]
+
+        // Create Datasource for CollectionView
+        locationSections = [
+            .campus(campusLocations),
+            .ctown(ctownLocations)
+        ]
+
+        setupConstraints()
+    }
+
+    private func setupConstraints() {
+        let backPadding = LayoutHelper.shared.getCustomVerticalPadding(size: 24)
+        let infoPadding = 3
+        let nextPadding = LayoutHelper.shared.getCustomVerticalPadding(size: 38)
+        let titlePadding = LayoutHelper.shared.getShortenedCustomVertPadding(size: 20)
+        let topPadding = LayoutHelper.shared.getShortenedCustomVertPadding(size: 92)
+
+        let collectionViewSize = CGSize(
+            width: LayoutHelper.shared.getCustomHoriztonalPadding(size: 312),
+            height: LayoutHelper.shared.getCustomVerticalPadding(size: 469)
+        )
+        let nextButtonSize = CGSize(
+            width: LayoutHelper.shared.getCustomVerticalPadding(size:175),
+            height: LayoutHelper.shared.getCustomVerticalPadding(size: 53)
+        )
+
+        titleLabel.snp.makeConstraints { make in
+            make.centerX.equalToSuperview()
+            make.top.equalTo(view.safeAreaLayoutGuide).inset(topPadding)
+        }
+
+        infoLabel.snp.makeConstraints { make in
+            make.centerX.equalToSuperview()
+            make.top.equalTo(titleLabel.snp.bottom).offset(infoPadding)
+        }
+
+        locationsCollectionView.snp.makeConstraints { make in
+            make.size.equalTo(collectionViewSize)
```

Comment:
to avoid using a specific size using the LayoutHelper, we should just set the bottom of `collectionView` to the top of the `nextButton` with some padding, like how we did for the onboarding screens and homeVC